src/main/java/de/caritas/cob/consultingtypeservice/api/consultingtypes/ConsultingTypeMongoRepositoryService.java

```diff
@@ -0,0 +1,89 @@
+package de.caritas.cob.consultingtypeservice.api.consultingtypes;
+
+import de.caritas.cob.consultingtypeservice.api.exception.UnexpectedErrorException;
+import de.caritas.cob.consultingtypeservice.api.exception.httpresponses.NotFoundException;
+import de.caritas.cob.consultingtypeservice.api.model.ConsultingTypeEntity;
+import de.caritas.cob.consultingtypeservice.api.service.LogService;
+import de.caritas.cob.consultingtypeservice.schemas.model.ConsultingType;
+import java.util.List;
+import java.util.Optional;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.BeanUtils;
+import org.springframework.stereotype.Repository;
+
+/**
+ * Repository for {@link ConsultingType}.
+ */
+@Repository("tenantUnaware")
+@RequiredArgsConstructor
+public class ConsultingTypeMongoRepositoryService implements ConsultingTypeRepositoryService {
+
+  private @NonNull ConsultingTypeRepository consultingTypeRepository;
```

Comment:
Could you please also declare these two beans as final?

Reply:
done